src/sass/modules/buttons.scss

```diff
@@ -126,11 +153,13 @@
 
   &:hover:enabled {
     background-color: color(blue, light);
+    color: $white;
```

Comment:
A little redundant, but needed in order to override the `.button-filter-toggle` and this seemed like a cleaner option than nesting `&.button-blue` inside `.button-filter-toggle`